src/sidebar/components/hooks/use-root-thread.js

```diff
@@ -1,6 +1,8 @@
 import useStore from '../../store/use-store';
 import threadAnnotations from '../../util/thread-annotations';
 
+import { useMemo } from 'preact/hooks';
```

Comment:
Can you move this import to the top of the file in a separate group from local imports.

Reply:
Good catch. I was moving fast at certain times yesterday, trying out different hooks and whatnot, and I forgot to clean this up. Perhaps we are getting closer to a point at which we can lint, in some way, for this.